src/redist/targets/GenerateMSIs.targets

```diff
@@ -393,7 +396,7 @@
       InstallerFile="$(CombinedFrameworkSdkHostMSIInstallerFile)"
       WixExtensions="WixBalExtension;WixUtilExtension;WixTagExtension"
       WixSrcFiles="@(BundleMsiWixSrcFiles)"
-      AdditionalBasePaths="$(MSBuildThisFileDirectory)packaging/windows/clisdk">
+      AdditionalBasePaths="$(MSBuildThisFileDirectory)packaging/windows/clisdk;$(PkgMicrosoft_DotNet_Build_Tasks_Installers)\build\wix\bundle">
```

Comment:
What is this property used for? There are 2 other references to this property in other `LightCommandPackageDrop` creations. Should those also provide `$(PkgMicrosoft_DotNet_Build_Tasks_Installers)\build\wix\bundle`?

Reply:
It modifies when/how we'd perform upgrades.

Reply:
Thank you for the explanations.